### ISSUE TITLE
Adding Vuetify to main.js breaks build

### ISSUE THREAD

**Issue by lobo-tuerto** [gridsome/gridsome#43]
Looks like I declared victory too early.

When trying to include Vuetify, I change `main.js` from this:

```js
import DefaultLayout from '~/layouts/Default.vue'

export default function (Vue) {
  Vue.component('Layout', DefaultLayout)
}
```

To this:

```js
import Vuetify from 'vuetify'
import DefaultLayout from '~/layouts/Default.vue'

import 'vuetify/dist/vuetify.min.css'

export default function (Vue) {
  Vue.use(Vuetify)
  Vue.component('Layout', DefaultLayout)
}
```

But right after that, if I build the default app with `yarn build`, it no longer works.
What's the right way to add Vuetify to Gridsome?

Cheers,
VA

P.S. This is part of a Gridsome tutorial I recently wrote at: https://lobotuerto.com/blog/quickstart-guide-for-a-new-gridsome-project/

**Comment by hjvedvik** [comment]
It clearly breaks something in the server-side renderer. But I got it working with this webpack config. I couldn't get the `VuetifyLoaderPlugin` to work, because it requires extra config APIs that Gridsome doesn't have yet. But I'll look into it :)

```js
const nodeExternals = require('webpack-node-externals')

module.exports = function (api) {
  api.chainWebpack((config, { isServer }) => {
    if (isServer) {
      config.externals([
        nodeExternals({
          whitelist: [/^vuetify/]
        })
      ])
    }
  })
}
```

**Comment by lobo-tuerto** [comment]
When trying to build with that config added, I see this:

```bash
$ yarn build
yarn run v1.12.3
warning ../../package.json: No license field
$ gridsome build
Gridsome v0.3.1

Initialize - 0.06s
Load sources - 0.04s
Create GraphQL schema - 0.03s
Generate code - 0.02s
Bootstrap finish - 0.15s
Run GraphQL (3 queries) - 0.06s
Compile assets - 9.99s                                                                                                      Failed to render /                      
/home/yomero/tmp/my-site/node_modules/gridsome/app/pages/404.vue:1
(function (exports, require, module, __filename, __dirname) { <template>
                                                              ^

SyntaxError: Unexpected token <
    at new Script (vm.js:80:7)
    at createScript (vm.js:264:10)
    at Object.runInThisContext (vm.js:316:10)
    at Module._compile (internal/modules/cjs/loader.js:670:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by hjvedvik** [comment]
Just published v0.3.2 which should be able to build your project.

**Comment by lobo-tuerto** [comment]
Closing this, as it's been fixed by https://github.com/gridsome/gridsome/commit/97e5e369b8d21cf9d40ac3a00a8432be1019a3ee
